index.html

```diff
@@ -153,7 +153,7 @@
 
         const draw = SVG('techtree').size(tree.width, tree.height)
 
-        draw.rect(tree.width, tree.height).attr({fill: 'none'}).click(hideHelp);
+        draw.rect(tree.width, tree.height).attr({fill: '#e7c28e00'}).click(hideHelp);
```

Comment:
I guess that works :sweat_smile:

Reply:
Yes! Setting the fill as none makes the rect not respond to clicks. Made the background fill with opacity as 0% to hide the color while still responding to clicks. 😄